src/zoid/buttons/util.js

```diff
@@ -121,6 +121,56 @@ export function getVenmoExperiment() : VenmoExperiment {
     }
 }
 
+export function createNoPaylaterExperiment() : Experiment | void {
+    const disableFunding = getDisableFunding();
+    const isDisableFundingPaylater = disableFunding && disableFunding.indexOf(FUNDING.PAYLATER) !== -1;
+    const enableFunding = getEnableFunding();
+    const isEnableFundingPaylater = enableFunding && enableFunding.indexOf(FUNDING.PAYLATER) !== -1;
+
+    const { paylater } = getFundingEligibility();
+    const isEligibleForPaylater = paylater?.eligible;
+    const isExperimentable = paylater?.products?.paylater?.variant === 'experimentable' || paylater?.products?.payIn4?.variant === 'experimentable';
+    // No experiment because ineligible, already forced on or off, unsupported browser
+    if (!isEligibleForPaylater
+        || !isExperimentable
+        || ((isDisableFundingPaylater || isEnableFundingPaylater) && isSupportedNativeBrowser())
+        || (isDevice() && !isSupportedNativeBrowser())
+    ) {
+        return;
+    }
+
+    if (isDevice()) {
+
+        if (isIos() && isSafari()) {
+            return createExperiment('disable_paylater_ios', 100);
+        }
+
+        if (isAndroid() && isChrome()) {
+            return createExperiment('disable_paylater_android', 100);
+        }
+    } else {
+        return createExperiment('disable_paylater_desktop', 100);
+    }
+}
+
+export function getNoPaylaterExperiment() : EligibilityExperiment {
+    const experiment = createNoPaylaterExperiment();
+
+    const disableFunding = getDisableFunding();
+    const isDisableFundingPaylater = disableFunding && disableFunding.indexOf(FUNDING.PAYLATER) !== -1;
+    const isNativeSupported = isSupportedNativeBrowser();
+    const isExperimentEnabled = experiment && experiment.isEnabled();
+    if (isDevice()) {
```

Comment:
Why is this logic here for PayLater?

Reply:
I did not realize that the native/desktop logic was unique to venmo and thought all buttons needed to be checked for supportedNative. I will remove if all native and desktop renders can be considered equivalent